src/utils/getThemeFunction.ts

```diff
@@ -0,0 +1,8 @@
+import { Theme, createTheme } from '@mui/material';
+import { themes } from '../themes/themes';
+
+export const getTheme = (targetThemeName: string): Theme => {
+  const theme = themes.find(({ themeName }) => themeName === targetThemeName)
+    || { themeName: '', themeObject: createTheme() };
+  return createTheme(theme.themeObject);
+};
```

Comment:
```suggestion
export const getTheme = (targetThemeName: string): Theme => {
  const theme = themes.find(({ themeName }) => themeName === targetThemeName)
    || { themeName: '', themeObject: createTheme() };
  
  return createTheme(theme.themeObject);
};
```

Always separate return statement by the blank line. Does the eslint haven't told you about this?